### PR TITLE
Scope call to list leo clusters

### PR DESCRIPTION
**Pull request by zarsky-broad**
Per @rtitle

### CODE REVIEW

**Review by elizzhao — APPROVED** [review]
looks good!

**Review by rtitle — APPROVED** [review]

File: src/libs/ajax.js [hunk 1]
```diff
@@ -645,8 +645,8 @@ const Methods = signal => ({
 
 
 const Jupyter = signal => ({
-  clustersList: async () => {
-    const res = await fetchLeo('api/clusters?saturnAutoCreated=true', _.mergeAll([authOpts(), appIdentifier, { signal }]))
+  clustersList: async project => {
+    const res = await fetchLeo(`api/clusters${project ? `/${project}` : ''}?saturnAutoCreated=true`, _.mergeAll([authOpts(), appIdentifier, { signal }]))
```
Comment (breilly2) [inline src/libs/ajax.js:649]: Should `project` be coerced into a boolean? Should work as is, so more for readability I think.

**Review by breilly2 — APPROVED** [review]
👍

**Review by katevoss — APPROVED** [review]

File: src/libs/ajax.js (hunk 1 above)
Comment (zarsky-broad) [reply src/libs/ajax.js:649]: For the moment I'm not going to worry about that, it's a valid point though

**Pull request merged**